cypress/integration/search_address.ts

```diff
@@ -0,0 +1,82 @@
+import { AuthRoles } from '../support/commands';
+
+describe('Use AddressLookup to search Hackney address api', () => {
+  it('using a building number & postcode', () => {
+    cy.visitAs(`/people/add`, AuthRoles.ChildrensGroup);
+    cy.get(`input[id=buildingNumber]`).click().type('1');
+    cy.get(`input[id=postcode]`).click().type('SW1A 0AA');
+    cy.get(`button[id=lookup-button]`).click();
+    cy.get(`select[id=address]`).should('be.visible');
+    cy.get(`select[id=address]`)
+      .select('THE SPEAKERS HOUSE, 1 PARLIAMENT SQUARE')
+      .should('have.text', 'THE SPEAKERS HOUSE, 1 PARLIAMENT SQUARE');
+  });
+
+  it('using just a postcode', () => {
+    cy.visitAs(`/people/add`, AuthRoles.ChildrensGroup);
+    cy.get(`input[id=postcode]`).click().type('SW1A 0AA');
+    cy.get(`button[id=lookup-button]`).click();
+    cy.get(`select[id=address]`).should('be.visible');
+    cy.get(`select[id=address]`)
+      .contains('THE SPEAKERS HOUSE, 1 PARLIAMENT SQUARE')
+      .should('have.length.at.least', 1);
+  });
+
+  it('using different building numbers & postcodes', () => {
+    cy.visitAs(`/people/add`, AuthRoles.ChildrensGroup);
+    cy.get(`input[id=buildingNumber]`).click().type('188');
+    cy.get(`input[id=postcode]`).click().type('SE17 1JJ');
+    cy.get(`button[id=lookup-button]`).click();
+    cy.get(`select[id=address]`).should('be.visible');
+    cy.get(`select[id=address]`)
+      .select('188 WALWORTH ROAD')
+      .should('have.text', '188 WALWORTH ROAD');
+
+    cy.get(`input[id=buildingNumber]`).click().clear().type('194');
+    cy.get(`input[id=postcode]`).click().clear().type('SE17 1JJ');
+    cy.get(`button[id=lookup-button]`).click();
+    cy.get(`select[id=address]`).should('be.visible');
+    cy.get(`select[id=address]`)
+      .select('194 WALWORTH ROAD')
+      .should('have.text', '194 WALWORTH ROAD');
+  });
+});
+
+describe('Validates user input', () => {
```

Comment:
These are good tests but they feel more like unit tests for the `AddressLookup` component. See https://github.com/LBHackney-IT/lbh-core-pathway-pilot/blob/ea9c13c6196bc0a2bcdfa77d2060e27d2deadbdd/components/ManagerApprovalDialog.test.tsx#L121-L137 as an example. Let me know if you want to 🍐 on dis.

Reply:
Thanks Ting, we've moved the validation tests into the AddressLookup.spec unit test file. 

We also tried moving the address search tests to the unit test file, however, this didn't work as we wanted to test the real response from the postcodeAPI and we were unable to make this axios.get() call in the unit tests without mocking the response, I hope this all makes sense